lvsfunc/comparison.py

```diff
@@ -605,8 +609,13 @@ def diff(*clips: vs.VideoNode,
     if (clips and len(clips) != 2) or (namedclips and len(namedclips) != 2):
         raise CustomValueError("Must pass exactly 2 `clips` or `namedclips`!", diff)
 
-    if not 1 <= thr < 128:
-        raise CustomValueError(f"`thr` must be between 1 and 128, not {thr}!", diff)
+    if not thr < 128:
+        raise CustomValueError(f"`thr` must be below 128!", diff)
```

Comment:
The error is missing the reason

Reply:
sorry, I don't understand

Reply:
You should specify why it errored, aka passing `reason=thr`.